SCClassLibrary/Common/Control/Server.sc

```diff
@@ -786,17 +786,19 @@ Server {
 		condition.wait
 	}
 
-	waitForBoot { |onComplete, limit = 100, onFailure, clock = \AppClock|
+	waitForBoot { |onComplete, limit = 100, onFailure, clock|
 		// onFailure.true: why is this necessary?
 		// this.boot also calls doWhenBooted.
 		// doWhenBooted prints the normal boot failure message.
 		// if the server fails to boot, the failure error gets posted TWICE.
 		// So, we suppress one of them.
+		clock = clock ?? { AppClock };
```

Comment:
this is not needed, because it is done in `doWhenBooted`.

Reply:
Thank you, and sorry. Done! https://github.com/supercollider/supercollider/pull/6198/commits/802e45051f7912dabae01d27345688ac4ecaf425